core/app/[locale]/(default)/account/[tab]/_components/account-tabs.tsx

```diff
@@ -1,20 +1,30 @@
 'use client';
 
 import { useTranslations } from 'next-intl';
-import { PropsWithChildren } from 'react';
+import { PropsWithChildren, useEffect } from 'react';
 
 import { Link } from '~/components/link';
 import { Tabs, TabsContent, TabsList, TabsTrigger } from '~/components/ui/tabs';
 
 import { TabType } from '../layout';
 
+import { useAccountStatusContext } from './account-status-provider';
+
 interface Props extends PropsWithChildren {
   tabs: TabType[];
   activeTab?: TabType;
 }
 
 export const AccountTabs = ({ children, activeTab, tabs }: Props) => {
   const t = useTranslations('Account.Home');
+  const context = useAccountStatusContext();
```

Comment:
why would not use destructuring here? what is a benefit of  keeping an additional variable?